cli/src/main/java/com/devonfw/tools/ide/context/GitContextImpl.java

```diff
@@ -292,6 +292,20 @@ IdeContext getContext() {
 
     return this.context;
   }
+
+  /**
+   * Checks if there is a git installation and throws an exception if there is none
+   */
+  private void verifyGitInstalled() {
+    String requiredBash = this.context.findBashRequired();
+    Path git = Path.of("git");
+    String binaryGitPath = this.context.getPath().findBinary(git).toString();
+    if (requiredBash == binaryGitPath) {
```

Comment:
Here you mixed two different checks:
1. verify that we have a bash. You do not need to assign the result of `this.context.findBashRequired()` to a variable since we only want to ensure that bash is available. Otherwise this method will throw an exception.
2. we want to check that git is installed, therefore you need to check `if (git == binaryGitPath) {`

Reply:
Seperated

Reply:
@WorkingAmeise thanks for your update. Now perfect 👍